test/k8sT/assertionHelpers.go

```diff
@@ -52,10 +52,10 @@ func ExpectCiliumOperatorReady(vm *helpers.Kubectl) {
 	ExpectWithOffset(1, err).Should(BeNil(), "Cilium operator was not able to get into ready state")
 }
 
-// ExpectCiliumRunning is a wrapper around helpers/WaitForPodsRunning. It
-// asserts the cilium pods are running on all nodes.
+// ExpectCiliumRunning is a wrapper around helpers/WaitForNPods. It
+// asserts the cilium pods are running on all nodes (but not yet ready!).
 func ExpectCiliumRunning(vm *helpers.Kubectl) {
-	err := vm.WaitforNPods(helpers.KubeSystemNamespace, "-l k8s-app=cilium", vm.GetNumNodes(), longTimeout)
+	err := vm.WaitforNPodsRunning(helpers.KubeSystemNamespace, "-l k8s-app=cilium", vm.GetNumNodes(), longTimeout)
```

Comment:
Should we add another check to wait for Cilium to be ready elsewhere in our CI? I'm wondering whether this change will introduce stability elsewhere.

Reply:
You mean something like `ExpectCiliumReady()`, which is the primary function used across the CI code? :)

`ExpectCiliumRunning()` is only used from the flannel path so it should be fine.

Reply:
(This was basically a regression introduced by some refactoring where someone didn't realise that for the flannel case we explicitly only want cilium to be running so that it deploys the CNI chaining configuration, then we can bootstrap flannel to provide the dataplane connectivity)

Reply:
Got it, thanks for clarifying - I did not check to see whether this function was utilized elsewhere. Ship it!